cl/search/models.py

```diff
@@ -2870,6 +2883,31 @@ class Meta:
         verbose_name_plural = "Opinion parentheticals"
 
 
+class ParentheticalGroup(models.Model):
+    opinion = models.ForeignKey(
+        Opinion, related_name="parenthetical_groups", on_delete=models.CASCADE
+    )
+    representative = models.ForeignKey(
+        Parenthetical,
+        related_name="represented_group",
+        on_delete=models.CASCADE,
+    )
```

Comment:
Could you please add help_text to these fields too? It took me a minute to figure out what the structure was and help_text would have helped.